frontend/src/pages/ActivityReport/Pages/activitySummary.js

```diff
@@ -343,7 +343,7 @@ const ActivitySummary = ({
         <div id="tta" />
         <div className="margin-top-2">
           <FormItem
-            label="What TTA was provided"
+            label="What type of TTA was provided?"
             name="ttaType"
```

Comment:
This was an additional request by Adam Roux to synchronize filter and form question language